_data/openseeds/ols-6/schedule.yaml

```diff
@@ -595,7 +595,7 @@ weeks:
       - slides: 
           https://drive.google.com/file/d/1w8AUnsbwSpqfEK0YxKmSZGIeIlEbppSN/view?usp=share_link
         speakers:
-        - gemmaturon
```

Comment:
Will do. 👍🏽

Reply:
Thanks a lot!!!!

Reply:
Okay, so far I have found something that's causing `Gemma` and `Fadanka` to be missing.

Gemma exists in [L4338](https://github.com/open-life-science/open-life-science.github.io/blob/801c7134c89721fc8cbea1f96b2fee81907fda1f/_data/people.yaml#L4338) and Fadanka in [L3993](https://github.com/open-life-science/open-life-science.github.io/blob/801c7134c89721fc8cbea1f96b2fee81907fda1f/_data/people.yaml#L3993) of the `people.yaml` file.

However, both individuals had their names updated in PR #907 to include their additional names:
- Gemma Turon **Rodrigo** (see [diff](https://github.com/open-life-science/open-life-science.github.io/pull/907/files#diff-81afb82f1179e89eb574f60daa8bed2d30f36d54bdc9672d5b308c4885acc3f3)) and 

<img width="1640" alt="Screenshot 2024-11-12 at 20 39 45" src="https://github.com/user-attachments/assets/2071c640-4e73-460e-b69f-327708818c77">

- Wapouo Fadanka Stephane (see [diff](https://github.com/open-life-science/open-life-science.github.io/pull/907/files#diff-81afb82f1179e89eb574f60daa8bed2d30f36d54bdc9672d5b308c4885acc3f3)).

<img width="1905" alt="Screenshot 2024-11-12 at 20 38 50" src="https://github.com/user-attachments/assets/8d041524-21b8-4285-8023-603329f5877f">

It no longer recognises [this Gemma Turon](https://docs.google.com/spreadsheets/d/13rJ_Qqre7VfiW6Ioh-v4r6Xc9PvodZYriJ1ZzUJrRF8/edit?gid=1181436982#gid=1181436982) as Gemma Turon Rodrigo.

**Solutions:**
- Update the Google Sheets or other data sources to match the new names.
- Use partial matching or regular expressions: This can become problematic if we have very similar names (e.g Umar Farouk Ahmad and Umar Ahmad).

Let me know what you think. 🙏

Reply:
The first solution is probably the best one